### ISSUE TITLE
Записки исследования

### ISSUE THREAD

**Issue by vshokorov** [Intelligent-Systems-Phystech/Shokorov-BS-Thesis#2]

**Comment by vshokorov** [comment]
## Работы по DDA за последнее время

1. [BDANN : BERT‑based domain adaptation neural network for multi‑modal fake news detection](https://dr.ntu.edu.sg/bitstream/10356/144285/2/BDANN%20BERT-Based%20Domain%20Adaptation%20Neural%20Network%20for%20Multi-Modal%20Fake%20News%20Detection.pdf)
2. [Generalized Source-free Domain Adaptation](https://arxiv.org/pdf/2108.01614.pdf)
<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/48180167/140024584-7570af78-a5e8-4210-a311-35f4471bd024.png)

простая идея, схожие объекты должны двигаться рядом при Source-free DA.
</details>

3. [Transformer-Based Source-Free Domain Adaptation](https://arxiv.org/pdf/2105.14138.pdf)
<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/48180167/140024721-b50c0db1-3fcc-4715-9441-b79ea3900515.png)
Техническая работа, прирост за счет архитектуры
</details>

4. [Transfer Learning and Deep Domain Adaptation](https://www.intechopen.com/chapters/73675)
    - Общий рассказ про то, где находится DA относительно других подходов. Есть некоторая теория, но больше на словах.
5. [Unsupervised Multi-source Domain Adaptation Without Access to Source Data](https://openaccess.thecvf.com/content/CVPR2021/papers/Ahmed_Unsupervised_Multi-Source_Domain_Adaptation_Without_Access_to_Source_Data_CVPR_2021_paper.pdf)
<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/48180167/140027776-113620a9-608d-42b6-84bb-02a53dc7753a.png)

- Делают псевдолейблинг по предсказанию различных предобученных классификаторв.
- и пользуются cluster assumption  in semi-supervised learning, которое предполагает, что границы принятия решений дискриминационной модели должны располагаться в областях входного пространства, которые не являются густонаселенными.
</details>

6. [A Prototype-Oriented Framework for Unsupervised Domain Adaptation](https://arxiv.org/pdf/2110.12024.pdf)
    - Много различных экспериментов, проверки предлагаемого подхода DA.
    - Не самая интересная теория, аккуратно размечают данные статистически ищут prototypes и т.д.
7. [Adversarial Domain Adaptation with Paired Examples for Acoustic Scene Classification on Different Recording Devices](https://arxiv.org/pdf/2110.09598.pdf)
    - Просто применили  CycleGAN для source & target датасетов, показали, что качество сильно выросло.
8. [Scaling Laws for the Few-Shot Adaptation of Pre-trained Image Classifiers](https://arxiv.org/pdf/2110.06990.pdf)
<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/48180167/140042754-0cb66031-4b31-4359-bffb-f5f948c37ae4.png)
</details>

9. [Uncertainty-aware Mean Teacher for Source-free Unsupervised Domain Adaptive 3D Object Detection](https://arxiv.org/pdf/2109.14651.pdf)
10. [Unsupervised Cross-Modality Domain Adaptation for Segmenting Vestibular Schwannoma and Cochlea with Data Augmentation and Model Ensemble](https://arxiv.org/pdf/2109.12169.pdf)
11. [Adaptive Sampling using POMDPs with Domain-Specific Considerations](https://arxiv.org/pdf/2109.11595.pdf)
12. [Source-Free Domain Adaptive Fundus Image Segmentation with Denoised Pseudo-Labeling](https://arxiv.org/pdf/2109.09735.pdf)
<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/48180167/140603631-fe5494ef-9a32-49fb-9415-4cd100a33cfd.png)
</details>

13. [Unsupervised Domain Adaptation with Semantic Consistency across Heterogeneous Modalities for MRI Prostate Lesion Segmentation](https://arxiv.org/pdf/2109.09736.pdf)
    - Подправили  CycleGAN, изменив немного Semantic cycle-consistency loss чтобы иметь возможность преобразовывать целевую переменную (сегментацию).
14. [Powering Comparative Classification with Sentiment Analysis via Domain Adaptive Knowledge Transfer](https://arxiv.org/pdf/2109.03819.pdf)
15. [DILBERT: Customized Pre-Training for Domain Adaptation withCategory Shift, with an Application to Aspect Extraction](https://arxiv.org/pdf/2109.00571.pdf)
16. [Learning Cross-modal Contrastive Features for Video Domain Adaptation](https://arxiv.org/pdf/2108.11974.pdf)
17. [Channel-Temporal Attention for First-Person Video Domain Adaptation](https://arxiv.org/pdf/2108.07846.pdf)
18. [DensePASS: Dense Panoramic Semantic Segmentation via Unsupervised Domain Adaptation with Attention-Augmented Context Exchange](https://arxiv.org/pdf/2108.06383.pdf)
19. [BiMaL: Bijective Maximum Likelihood Approach to Domain Adaptation in Semantic Scene Segmentation](https://arxiv.org/pdf/2108.03267.pdf)
20. [HierMUD: Hierarchical Multi-task Unsupervised Domain Adaptation between Bridges for Drive-by Damage Diagnosis
](https://arxiv.org/pdf/2107.11435.pdf)
21. ⭐️ [CLDA: Contrastive Learning for Semi-Supervised Domain Adaptation](https://arxiv.org/pdf/2107.00085.pdf)
22. [Domain adaptation for person re-identification on new unlabeled data using AlignedReID++](https://arxiv.org/pdf/2106.15693.pdf)
23. [Multi-Task Learning and Adapted Knowledge Models for Emotion-Cause Extraction](https://arxiv.org/pdf/2106.09790.pdf)
24. ⭐️ [Predicting the Success of Domain Adaptation in Text Similarity](https://arxiv.org/pdf/2106.04641.pdf)
25. ⭐️ [Correlated Adversarial Joint Discrepancy Adaptation Network](https://arxiv.org/pdf/2105.08808.pdf)
26. ⭐️ [PixMatch: Unsupervised Domain Adaptation via Pixelwise Consistency Training](https://arxiv.org/pdf/2105.08128.pdf)
27. [Incremental Multi-Target Domain Adaptation for Object Detection with Efficient Domain Transfer](https://arxiv.org/pdf/2104.06476.pdf)

**Comment by vshokorov** [comment]
еще теоретическая работа 2006г. [Analysis of Representations for Domain Adaptation](https://papers.nips.cc/paper/2006/file/b1b0432ceafb0ce714426e9114852ac7-Paper.pdf)
